inc/admin.php

```diff
@@ -173,10 +172,33 @@ function save_post( $post_id, $post ) {
 
 		if ( ! empty( $panels_data['widgets'] ) || ! empty( $panels_data['grids'] ) ) {
 			update_post_meta( $post_id, 'panels_data', map_deep( $panels_data, array( 'SiteOrigin_Panels_Admin', 'double_slash_string' ) ) );
+
+			if( siteorigin_panels_setting( 'copy-content' ) ) {
+				// Save a copy of all this content into the posts's post_content
+				$GLOBALS[ 'SITEORIGIN_PANELS_DATABASE_RENDER' ] = true;
+
+				// This gives other plugins the chance to
+				do_action( 'siteorigin_panels_setup_database_render', $post_id );
+				SiteOrigin_Panels_Post_Content::single();
```

Comment:
It's unclear what this does without going to look at the implementation of `SiteOrigin_Panels_Post_Content`. I'd consider renaming it to something like `SiteOrigin_Panels_Post_Content_Filters` or the bigger mouthful `SiteOrigin_Panels_Post_Content_Data_Adapter`. :stuck_out_tongue: And perhaps go a step further and make the call to add the filters an explicit method, instead of tucked away in the constructor.